### DISCUSSION TITLE
Logging configuration in docker-compose

### DISCUSSION THREAD

**Discussion by frblazquez** [strimzi/strimzi-kafka-operator#5616] [Q&A]
[Stack Overflow](https://stackoverflow.com/questions/69332921/strimzi-kafka-docker-compose-logging-configuration): which is the proper way to configure logging in a ***docker-compose.yaml*** for the images in [docker hub](https://hub.docker.com/r/strimzi/kafka)?

**Comment by scholzj** [comment]
Strimzi is not designed for running in Docker Compose. It focuses on running Apache Kafka on Kubernetes using the operator pattern. So there is no defined way how to configure logging there.
